lib/bloc/fee_options/fee_options_bloc.dart

```diff
@@ -78,10 +78,8 @@ class FeeOptionsBloc extends Cubit<FeeOptionsState> {
   }
 
   Future<int> _calculateTransactionFee(String address, int satsPerVbyte) async {
-    final response = await _breezLib.prepareSweep(
-      address: address,
-      satsPerVbyte: satsPerVbyte,
-    );
+    final req = PrepareSweepRequest(satPerVbyte: satsPerVbyte, toAddress: address);
```

Comment:
Rename address field to toAddress and add a comma afterwards for indentation.

Reply:
fixed